docs/README.md

```diff
@@ -1,58 +1,57 @@
-# VTEX React App Template
+First of all (even before the block's name), its README.md should contain the following sentence at the very beginning:
 
-Our guide repository to structure for react apps, that should be used as a template.
+`📢 Use this project, [contribute](https://github.com/{OrganizationName/{AppName}) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).`
```

Comment:
Where should this url `https://github.com/{OrganizationName/{AppName}` take the user to?